### PR TITLE
feat: add csl statusfy integration in emergency. refs #1211

### PR DESCRIPTION
**Pull request by dianalin2**

### CODE REVIEW

File: intranet/apps/emerg/views.py [hunk 1]
```diff
@@ -68,12 +68,30 @@ def check_emerg():
...
+        r = requests.get(settings.CSL_STATUS_PAGE, timeout=timeout)
+    except requests.exceptions.Timeout:
+        pass
+
+    try:
+        csl_status = json.loads(r.text)
+    except json.decoder.JSONDecodeError:
+        return False, None
+
+    for system in csl_status['systems']:
```
Comment (smayya337) [inline intranet/apps/emerg/views.py:83]: I think that with the code that's currently here, the status isn't updated if there's a  CSL issue, so you'll never see a CSL issue unless there's an FCPS emergency announcement as well. Having `status` update whenever there's a CSL issue will fix that. Also, we probably don't want informational updates to be in the yellow (e.g. current Borg cluster maintenance, so we can filter that out), and we shouldn't have multiple instances of the same thing in the `announcements` list (the JSON has the same issue under each affected system).

With all that said, here's how I worked around all of this:
```py
status = False
    for system in csl_status['systems']:
        if system['status'] != 'ok':
            status = True
            issues = system['unresolvedIssues']
            for issue in issues:
                desc = requests.get(issue['permalink']).text
                soup = BeautifulSoup(desc, "html.parser")
                desc = soup.find_all('p')[1].text
                val = {"title": issue['title'], "body": desc}
                if val not in announcements and issue['severity'] != 'notice':
                    announcements.append(val)
```

**Review by smayya337 — CHANGES_REQUESTED** [review]
I noticed one issue that prevented the CSL status updates from showing up on my end, as well as a few places to tweak how everything is displayed. Besides that, the only other thing I can think of is maybe removing the "-FCPS" in the bottom right corner of the announcement area, and this should be good (though I defer to @Laur04 there).

**Review by Laur04 — CHANGES_REQUESTED** [review]
Changes look fine to me.

@smayya337 what prevents this from pulling even the "All Good" status from the status page?

@dianalin2 fix CI, then you're good.

**Comment by Laur04** [comment]
@dianalin2 you've got merge conflicts now. `dev` was updated recently, so rebase against it.

**Comment by Laur04** [comment]
@dianalin2 don't worry about Codacy

**Review by Laur04 — CHANGES_REQUESTED** [review]
LGTM - squash commits and I'll merge

**Comment by Laur04** [comment]
@dianalin2 I didn't see it this time, so my bad, but in the future, please remove the old commit messages from the squashed commit message.

**Pull request merged**